File: 04 - ORIENTAÇÃO A OBJETOS/nodeJS - JavaScript_/Exercicio 3/Com Pacotes/Pessoa.js
module.exports = class Pessoa {
    constructor(nome, idade, peso){
        this.nome = nome
        this.idade = idade
        this.peso = peso
    }

    dizerInfo(){
        return `${this.nome} idade: ${this.idade}, peso: ${this.peso}`
    }

    dizerNome(){
        return this.nome
    }
}

File: 04-ORIENTACAO_A_OBJETOS/Java/Exercicio 3/Pessoa.java
/*
TestePessoa.java
 */
public class Pessoa {
    String nome;
    int idade;
    double peso;
    
    //Construtor 1 (Passando todos atributos) - pe1
    Pessoa(String nome, int idade, double peso){
        this.nome = nome;
        this.peso = peso;
        this.idade = idade;
    }
    
    //Construtor 2 (Passando apenas um atributo) - pe2
    Pessoa(String nome){
        this.nome = nome;
    }
    
    String dizerInfo(){
        return this.nome+" idade: "+this.idade+" - peso: "+this.peso;
    }
    
    String dizerInfoNome(){
        return this.nome;
    }
}
